### DISCUSSION TITLE
چگونه لینکی که هایدیفای جنریت میکند را تغییر بدیم؟

### DISCUSSION THREAD

**Discussion by lostsoul6** [hiddify/Hiddify-Manager#433] [Q&A سوالات رایج]
سلام دوستان . برای اولین بار هایدیفای رو نصب کردم روی یه سرور و لینکی جنریت کرده برام با فرمت زیر :  

`vless://c7adbae1-ea5b-4ecb-845c-63250ab79545@somedomain:443?security=tls&sni=sub.dom.com&type=grpc&alpn=http/1.1&path=YdU2DQtAREBBc8jznQocxa0PMo&host=sub.dom.com&serviceName=YdU2DQtAREBBc8jznQocxa0PMo&mode=multi&encryption=none&fp=chrome&headerType=None#tls_grpc_CDN_vless_FR1`

توی این لینک یه سری پارامتر هست که من زمانی که با x-ui کانفیگ میسازم ، اینا رو نمیزنم و مشکلی هم ندارم . مثلا alpn= یا mode=multi یا fp=chrome و headertype=none خواستم بدونم چطور میشه لینکی که ساخته میشه رو کاری کنیم اون چیزایی که ما میخوایم رو فقط بزنه .

اگه فایلی چیزی هست داخل سرور که میشه syntax رو تغییر داد برای ساخت لینک لطفا اطلاع بدید کدوم فایله .

یه سوال دیگه هم دارم . محاسبه حجم پنل هایدیفای به چه صورته ؟ آیا اگر لینک رو تغییر بدم اون بخش به مشکل نمیخوره ؟ خودم حدس میزنم مشکل نخوره چون فکر میکنم با UUID اونش محاسبه میشه .

تشکر .

**Comment by lostsoul6** [comment]
و اینکه یه سوال دیگه . این پورتی که توی لینک بالا هست الان 443 هست . چطوری میشه عددش رو خودمون برای هر کانفیگ تعریف کنیم ؟ مثلا 2083 یا 2096 و 8443 بزنیم ؟

**Comment by bamdad23** [comment]
RQ رو که اسکن میکنم تو اپ fari vpn
میگه لینک سایت مورد نظر ساپورت نمیشه. کسی میتونه توضیح بده؟ با سپاس
